packages/core/test/unit/dialects/sqlite/query-generator.test.js

```diff
@@ -382,6 +382,23 @@ if (dialect === 'sqlite') {
           expectation: 'PRAGMA foreign_key_check(`schema.myTable`);',
         },
       ],
+      describeCreateTableQuery: [
```

Comment:
We're moving tests away from this file and making dedicated tests files for each QueryGenerator function here; https://github.com/sequelize/sequelize/tree/main/packages/core/test/unit/query-generator

Please add a file for `describeCreateTableQuery` as well, you can i.e. check out https://github.com/sequelize/sequelize/blob/main/packages/core/test/unit/query-generator/describe-table-query.test.ts for the different test scenarios